.changeset/four-lobsters-lay.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': major
```

Comment:
Does it require major release?

Reply:
I initially thought that new changes were to be labeled 'major' due to a major release coming up, but after reading through Semver documentation, it seems that a 'patch' label would be more accurate, since the changes I made are most closely described as a bug fix.

Reply:
changeset no longer needed